### ISSUE TITLE
clicking instagram - open into new tab

### ISSUE THREAD

**Issue by ashleysidmore** [RileyHayesMpls/RileyHayes2.0#21]
can we change this so clicking into instagram loads instagram on a different tab?

**Comment by darsico** [comment]
Do we want this link to open in a new tab when clicked? @ashleysidmore 
<img width="212" alt="image" src="https://github.com/user-attachments/assets/8128d40f-66c0-49ba-92d5-11f5259467d5">

**Comment by ashleysidmore** [comment]
yes, as well as the linkedin as well and when clicking on the instagram feed should open a new tab

**Comment by darsico** [comment]
i just made the updates and should be good to go!

**Comment by ashleysidmore** [comment]
The delta case study still opens a new tab. Can you fix that?

**Comment by darsico** [comment]
Done!
